### PR TITLE
Functional tests - Add new test 'enable/disable new product page'

### PR DESCRIPTION
**Pull request by nesrineabdmouleh**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Add new test 'enable/disable new product page'
| Type?             | refacto
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | no need
| Fixed ticket?     | no
| Related PRs       | no
| Sponsor company   | no

### CODE REVIEW

**Review by boubkerbribri — DISMISSED** [review]

File: tests/UI/campaigns/functional/BO/14_advancedParameters/11_newAndExperimentalFeatures/01_enableDisableExperimentalProductV2.ts [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+      const pageTitle: string = await productsPageV1.getPageTitle(page);
+      await expect(pageTitle).to.contains(productsPageV1.pageTitle);
+    });
+
+    it('should go to Add product page', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'goToAddProductV1Page', baseContext);
+
+      await productsPageV1.goToAddProductPage(page);
+
+      const createProductTitle: string = await addProductPage.getPageTitle(page);
```
Comment (Progi1984) [inline tests/UI/campaigns/functional/BO/14_advancedParameters/11_newAndExperimentalFeatures/01_enableDisableExperimentalProductV2.ts]:
```suggestion
      const createProductTitle = await addProductPage.getPageTitle(page);
```

File: tests/UI/campaigns/functional/BO/14_advancedParameters/11_newAndExperimentalFeatures/01_enableDisableExperimentalProductV2.ts [hunk 2]
```diff
@@ -0,0 +1,143 @@
...
+      await testContext.addContextItem(this, 'testIdentifier', 'goToProductsV1Page', baseContext);
+
+      await dashboardPage.goToSubMenu(
+        page,
+        dashboardPage.catalogParentLink,
+        dashboardPage.productsLink,
+      );
+      await productsPageV1.closeSfToolBar(page);
+
+      const pageTitle: string = await productsPageV1.getPageTitle(page);
```
Comment (Progi1984) [inline tests/UI/campaigns/functional/BO/14_advancedParameters/11_newAndExperimentalFeatures/01_enableDisableExperimentalProductV2.ts]:
```suggestion
      const pageTitle = await productsPageV1.getPageTitle(page);
```

File: tests/UI/campaigns/functional/BO/14_advancedParameters/11_newAndExperimentalFeatures/01_enableDisableExperimentalProductV2.ts [hunk 3]
```diff
@@ -0,0 +1,143 @@
...
+      await expect(isModalVisible).to.be.true;
+    });
+
+    it('should choose \'Standard product\'', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'chooseStandardProduct', baseContext);
+
+      await productsPageV2.selectProductType(page, 'standard');
+      await productsPageV2.clickOnAddNewProduct(page);
+
+      const pageTitle: string = await createProductsPageV2.getPageTitle(page);
```
Comment (Progi1984) [inline tests/UI/campaigns/functional/BO/14_advancedParameters/11_newAndExperimentalFeatures/01_enableDisableExperimentalProductV2.ts]:
```suggestion
      const pageTitle = await createProductsPageV2.getPageTitle(page);
```

File: tests/UI/campaigns/functional/BO/14_advancedParameters/11_newAndExperimentalFeatures/01_enableDisableExperimentalProductV2.ts [hunk 4]
```diff
@@ -0,0 +1,143 @@
...
+      await productsPageV2.closeSfToolBar(page);
+
+      const pageTitle: string = await productsPageV2.getPageTitle(page);
+      await expect(pageTitle).to.contains(productsPageV2.pageTitle);
+    });
+
+    it('should click on \'New product\' button and check new product modal', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'clickOnNewProductButton', baseContext);
+
+      const isModalVisible: boolean = await productsPageV2.clickOnNewProductButton(page);
```
Comment (Progi1984) [inline tests/UI/campaigns/functional/BO/14_advancedParameters/11_newAndExperimentalFeatures/01_enableDisableExperimentalProductV2.ts]:
```suggestion
      const isModalVisible = await productsPageV2.clickOnNewProductButton(page);
```

File: tests/UI/campaigns/functional/BO/14_advancedParameters/11_newAndExperimentalFeatures/01_enableDisableExperimentalProductV2.ts [hunk 5]
```diff
@@ -0,0 +1,143 @@
...
+      await testContext.addContextItem(this, 'testIdentifier', 'goToProductsPageV2', baseContext);
+
+      await dashboardPage.goToSubMenu(
+        page,
+        dashboardPage.catalogParentLink,
+        dashboardPage.productsLink,
+      );
+      await productsPageV2.closeSfToolBar(page);
+
+      const pageTitle: string = await productsPageV2.getPageTitle(page);
```
Comment (Progi1984) [inline tests/UI/campaigns/functional/BO/14_advancedParameters/11_newAndExperimentalFeatures/01_enableDisableExperimentalProductV2.ts]:
```suggestion
      const pageTitle = await productsPageV2.getPageTitle(page);
```

File: tests/UI/campaigns/functional/BO/14_advancedParameters/11_newAndExperimentalFeatures/01_enableDisableExperimentalProductV2.ts [hunk 6]
```diff
@@ -0,0 +1,143 @@
...
+      await featureFlagPage.closeSfToolBar(page);
+
+      const pageTitle: string = await featureFlagPage.getPageTitle(page);
+      await expect(pageTitle).to.contains(featureFlagPage.pageTitle);
+    });
+
+    it('should enable New product page - Single store', async function () {
+      await testContext.addContextItem(this, 'testIdentifier', 'enableNewProductPage', baseContext);
+
+      const successMessage: string = await featureFlagPage.setNewProductPage(page, true);
```
Comment (Progi1984) [inline tests/UI/campaigns/functional/BO/14_advancedParameters/11_newAndExperimentalFeatures/01_enableDisableExperimentalProductV2.ts]:
```suggestion
      const successMessage = await featureFlagPage.setNewProductPage(page, true);
```

File: tests/UI/campaigns/functional/BO/14_advancedParameters/11_newAndExperimentalFeatures/01_enableDisableExperimentalProductV2.ts [hunk 7]
```diff
@@ -0,0 +1,143 @@
...
+      await testContext.addContextItem(this, 'testIdentifier', 'goToFeatureFlagPage', baseContext);
+
+      await dashboardPage.goToSubMenu(
+        page,
+        dashboardPage.advancedParametersLink,
+        dashboardPage.featureFlagLink,
+      );
+      await featureFlagPage.closeSfToolBar(page);
+
+      const pageTitle: string = await featureFlagPage.getPageTitle(page);
```
Comment (Progi1984) [inline tests/UI/campaigns/functional/BO/14_advancedParameters/11_newAndExperimentalFeatures/01_enableDisableExperimentalProductV2.ts]:
```suggestion
      const pageTitle = await featureFlagPage.getPageTitle(page);
```

**Review by Progi1984 — COMMENTED** [review]
Small feedback about type inference.

File: tests/UI/campaigns/functional/BO/14_advancedParameters/11_newAndExperimentalFeatures/01_enableDisableExperimentalProductV2.ts (hunk 7 above)
Comment (nesrineabdmouleh) [reply tests/UI/campaigns/functional/BO/14_advancedParameters/11_newAndExperimentalFeatures/01_enableDisableExperimentalProductV2.ts]: Done thx :+1:

**Review by Progi1984 — APPROVED** [review]

**Review by boubkerbribri — APPROVED** [review]

**Comment by boubkerbribri** [comment]
Thanks @nesrineabdmouleh

**Pull request merged**